spades/3.6.0/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         spades
+# Software Version: 3.6.0
+# Description:      Tools (written in C using htslib) for manipulating next-generation sequencing data
+# Website:          http://bioinf.spbau.ru/spades
+# Tags:             Genomics
+# Provides:         spades 3.6.0
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/spades 3.6.0/.
+# Pull Cmd:         docker pull biodckrdev/spades
+# Run Cmd:          docker run --rm -it biodckrdev/spades <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)